src/tir/schedule/primitive/loop_transformation.cc

```diff
@@ -385,6 +511,122 @@ StmtSRef Fuse(ScheduleState self, const Array<StmtSRef>& loop_srefs) {
   return self->stmt2ref.at(new_stmt.get());
 }
 
+void Reorder(ScheduleState self, const Array<StmtSRef>& ordered_loop_srefs) {
+  if (ordered_loop_srefs.empty() || ordered_loop_srefs.size() == 1) {
```

Comment:
to make it more concise:

```suggestion
  if (ordered_loop_srefs.size() <= 1) {
```